src/NetworkViewUi.lua

```diff
@@ -132,17 +139,37 @@ function M.get_list_of_items(view_type)
   local items = {}
 
   if view_type == "item" then
+    local missing = GlobalState.missing_item_filter()
```

Comment:
Why did you keep missing items on the existing pages rather than make a new page just for missing items?

Reply:
I thought having just two tabs (items and fluids) might be easier to use.
Also, I didn't really understand the GUI stuff at that point.
But I'd be OK with a new tab and combining shortages from all sources.